.github/workflows/tests.yaml

```diff
@@ -59,9 +59,38 @@ jobs:
     - name: Run PHPCS Code Style Checker
       run: dev/sh/style
 
+  docfx:
+    name: "Run DocFX tests"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
```

Comment:
nit: 3.11 is the latest but it's beta/untested (releasing next month) - 3.9 (stable, tested) or 3.10 (stable, slightly less tested throughout googleapis) could be better to test this against than 3.8. The oldest supported version at the moment is 3.7, so 3.8 will be soon. But, fine to leave as is for this PR.